### DISCUSSION TITLE
How to find data for each region of  interest

### DISCUSSION THREAD

**Discussion by mori1202-afk** [DeniseCaiLab/ezTrack#87] [Q&A]
Thanks for developing a wonderful app!
I would like to obtain the distance traveled and time spent for each region of interest throughout the test, is this possible? 
Perhaps this issue comes from me not being able to read the Summarystatus.csv file correctly. 
What do the numbers in the cells under each ROI mean? Is it the ratio of time spent in the ROI when the observation time is set as 1? If so, is there a ratio of distance traveled?

**Comment by ZachPenn** [comment]
Hi there.  The summary stats file that is output gives the proportion of time in each ROI.  It does not provide the distance travelled for each ROI.  However, it's quite simple to calculate.   You'll need to use file ending in 'LocationOutput', which gives the distance travelled per frame and the ROI the animal is in for each frame.  If you sort the spreadsheet/dataframe by the ROI the animal is in and then sum the distance travelled for each region you'll be able to get the total distance travelled.
